packages/design-system-twrnc-preset/package.json

```diff
@@ -67,9 +67,8 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "react": "^18.2.0",
-    "react-native": "0.72.15",
-    "twrnc": "^4.5.1"
+    "@metamask/design-tokens": "^7.1.0",
+    "react": "^18.2.0"
```

Comment:
Removing react-native since it’s unused in this package. Moved design tokens to peerDependencies to ensure only a single version is used by the consuming package.